src/module/classTask.js

```diff
@@ -0,0 +1,9 @@
+class Task {
+  constructor(code, description, completed) {
+    this.code = code;
```

Comment:
- [ ] [OPTIONAL] I highly suggest that you use `index` instead of `code` it's self-descriptive